### DISCUSSION TITLE
create siblings fails

### DISCUSSION THREAD

**Discussion by leolivier** [ryanhowdy/fcms#659] [General]
I tested the family tree and was able to create people, then children, spouse, ... but siblings systematically fails with this screen:
![image](https://github.com/ryanhowdy/fcms/assets/4014752/4e313aa9-a052-4a31-92d9-253b8a723ae8)

When I hit the back button, the sibling is not created

**Comment by ryanhowdy** [comment]
Correct, this functionality has not been added yet.  It's just some code stubs to help get started, when I go back to the family tree.  It is missing many more features, not just sibling.

**Reply by leolivier** [reply]
Hi Ryan
I broke my leg quite badly 2 weeks ago so I'm on sick leave for several weeks and I can help although I 'm not familiar with Laravel (I know Php).
Tell me if you 're interested and on what topic you think I can help
